inc/Engine/Optimization/Minify/JS/Combine.php

```diff
@@ -408,6 +408,9 @@ protected function add_to_minify( $content ) {
 	 */
 	protected function get_excluded_inline_content() {
 		$defaults = [
+			'RocketBrowserCompatibilityChecker',
+			'RocketLazyLoadScripts',
+			'RocketPreloadLinks',
```

Comment:
The `RocketLazyLoadScripts` and `RocketPreloadLinks` variable names are not present in the minified scripts as we use a mangler in the gulp task to further reduce the bytes in each of the scripts.

However, we do retain the `RocketBrowserCompatibilityChecker` name. All 3 of these scripts have that variable name in them. Therefore, we can simplify this to just `RocketBrowserCompatibilityChecker`.